src/components/QueryEditor/VisualQueryEditor.tsx

```diff
@@ -92,6 +93,9 @@ export const VisualQueryEditor: React.FC<VisualQueryEditorProps> = (props) => {
           </EditorField>
         </EditorFieldGroup>
       </EditorRow>
+      <FilterSection {...props} tableSchema={tableSchema} />
+      {/* TODO: Use proper preview component */}
+      <pre>{query.query}</pre>
```

Comment:
I plan to add a proper component for this in the future, but in the meantime, this is useful for devs purposes.